packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -536,6 +494,7 @@ describe('Incoming Payment Service', (): void => {
         })
         assert.ok(!isIncomingPaymentError(incomingPaymentOrError))
         incomingPayment = incomingPaymentOrError
+        await accountingService.createLiquidityAccount(incomingPayment)
```

Comment:
This `beforeEach` could be updated to just call [`createIncomingPayment`](https://github.com/interledger/rafiki/blob/main/packages/backend/src/tests/incomingPayment.ts#L9)